packages/starlight/utils/route-data.ts

```diff
@@ -58,7 +56,6 @@ export function generateRouteData({
 		toc: getToC(props),
 		lastUpdated: getLastUpdated(props),
 		editUrl: getEditUrl(props),
-		labels: useTranslations(locale).all(),
 	};
```

Comment:
Then we can include that proxy in the route data:

```suggestion
		labels,
	};
```